charts/cni-metrics-helper/values.yaml

```diff
@@ -34,3 +34,6 @@ revisionHistoryLimit: 10
 podSecurityContext: {}
 
 containerSecurityContext: {}
+
+deployment:
```

Comment:
nit: let's remove the `deployment` parent and just use `podAnnotations`, same as VPC CNI